tests/rbd_mirror/test_rbd_mirror_peer_id.py

```diff
@@ -0,0 +1,212 @@
+       and "health: WARNING" on cluster A
+    7. Reset correct client id to mirror peer client using
+        "rbd mirror pool peer set <peer UUID from step 4> client client.rbd-mirror-peer" command on cluster B
+    8. Wait 30-90 seconds and verify that "rbd mirror pool status" reports "health: OK" on both clusters again
+    9. Grab service_id and instance_id from "rbd mirror pool status --verbose" output on cluster B again
+    10. Verify that service_id from step 3 is equal to the one from step 9
+    11. Verify that instance_id from step 3 is not same than the one from step 9
+    8. Perform test steps for both Replicated and EC pool
+"""
+
+import json
+import time
+from copy import deepcopy
+
+from ceph.rbd.initial_config import initial_mirror_config
+from ceph.rbd.utils import getdict
+from ceph.rbd.workflows.cleanup import cleanup
+from ceph.rbd.workflows.rbd_mirror import check_pool_mirror_status
+from utility.log import Log
+
+log = Log(__name__)
+
+
+def test_peer_ID(mirror1, mirror2, primary_client, secondary_client, pool_type, **kw):
+    """Method to validate Change and Reset peer ID
+    and validate the mirror pool status reflects the same
+
+    Args:
+        mirror1: RBD Mirror primary cluster Object
+        mirror2: RBD Mirror secondary cluster Object
+        primary_client: RBD Mirror primary cluster client node
+        secondary_client: RBD Mirror secondary cluster client node
+        pool_type: Pool type (replicated or EC)
+        **kw: test data
+    Returns:
+        int: The return value. 0 for success, 1 otherwise
+    """
+    try:
+        log.info("Starting snapshot RBD mirroring test case")
+        config = deepcopy(kw.get("config").get(pool_type))
+
+        for pool_name, pool_config in getdict(config).items():
+            multi_image_config = getdict(pool_config)
+
+            for image, image_config in multi_image_config.items():
+                # Verify that "rbd mirror pool status" reports "health: OK"on primary cluster
+                if check_pool_mirror_status(mirror1, pool_name, "Primary_site", "OK"):
+                    return 1
+                log.info("Primary site health is OK")
+
+                # Verify that "rbd mirror pool status" reports "health: OK"on secondary cluster
+                if check_pool_mirror_status(mirror2, pool_name, "Secondary_site", "OK"):
+                    return 1
+                log.info("Secondary site health is OK")
+
+                # Grab service_id and instance_id from secondary cluster
+                pool_config = {"pool": pool_name, "verbose": True, "format": "json"}
+                output, err = mirror2.mirror.pool.status(**pool_config)
+                pool_status = json.loads(output)
+                service_id = pool_status["daemons"][0]["service_id"]
+                instance_id = pool_status["daemons"][0]["instance_id"]
+
+                log.info(f"Service ID: {service_id}")
```

Comment:
When you have mentioned Primary cluster or secondary cluster in every log of yours, it would make sense to mention that the service id and instance id are from secondary cluster here right? Instead of plainly mentioning service id and instance id

Reply:
done